tests/integration-e2e/test_coreneuron_directmode.py

```diff
@@ -1,17 +1,27 @@
 import os
 import numpy.testing as npt
 import numpy as np
+import pytest
+from pathlib import Path
 
 
-def test_coreneuron_no_write_model(USECASE3):
+SIM_DIR = Path(__file__).parent.parent.absolute() / "simulations"
+
+
+@pytest.mark.parametrize("create_tmp_simulation_file", [
+    {
+        "src_dir": str(SIM_DIR / "usecase3"),
+        "simconfig_file": "simulation_sonata_coreneuron.json"
+    }
+], indirect=True)
+def test_coreneuron_no_write_model(create_tmp_simulation_file):
     from libsonata import SpikeReader, ElementReportReader
```

Comment:
is it required that these are loaded late? Are we breaking a circular dependency or something?